ui/src/components/auth/Login.vue

```diff
@@ -72,19 +72,18 @@ function handleSubmit(): void {
           loggedInUser.avatar = Avatars[0];
         }
         userStore.updateCurrentUser(loggedInUser);
-        authStore.updateRegisteredUsername(null);
+        authStore.updateRegisteredUsername("");
         Swal.fire({
           icon: "success",
           title: "Success",
           text: "Login successful"
         }).then(() => {
           userStore.clearOptionalCookies();
-          if (previousAppUrl.value) {
-            window.location.href = previousAppUrl.value;
+          if (authReturnUrl.value) {
+            window.location.href = authReturnUrl.value;
```

Comment:
was preventing the above routings from occuring, keeping the page on login indefinitely. What the the functionality for this meant to be?